ocaml/idl/datamodel_lifecycle.ml

```diff
@@ -21,6 +21,14 @@ let prototyped_of_field = function
       Some "23.14.0"
   | "Repository", "gpgkey_path" ->
       Some "22.12.0"
+  | "Cluster", "total_votes" ->
+      Some "23.32.0-next"
+  | "Cluster", "quorum" ->
+      Some "23.32.0"
```

Comment:
Was this added previously? otherwise it should be 
```suggestion
      Some "23.32.0-next"
```

Reply:
yes, you are right, should be next